build_debian.sh

```diff
@@ -217,7 +217,9 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     unzip                   \
     gdisk                   \
     sysfsutils              \
-    grub2-common
+    grub2-common            \
+    dosfstools              \
+    parted
```

Comment:
why do we need to install dosfstools, parted in the base image?

Reply:
separate PR due to different purpose.

Reply:
These tools are needed in order to use a script (post migration) to revive free blocks and restore the OS9 partitions. I can remove them here since these can be installed on need basis on relevant units.